metamorph/server.go

```diff
@@ -279,10 +279,6 @@ func (s *Server) PutTransactions(ctx context.Context, req *metamorph_api.Transac
 			RawTx:         txReq.RawTx,
 		}
 
-		if err = s.processor.Set(ctx, &ProcessorRequest{Data: sReq}); err != nil {
```

Comment:
what is this all about ?

Reply:
I found that this setting is not needed as it already happens here: https://github.com/bitcoin-sv/arc/blob/f0c4abc631fcba488ba80df889496694f3bf8982/metamorph/processor.go#L529

With badger implementation the setting twice didn't raise an error, but postgres complained about the duplicate primary key, that is how I noticed it.